src/main/java/io/netty/incubator/codec/http3/CharSequenceMap.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.incubator.codec.http3;
+
+import io.netty.handler.codec.DefaultHeaders;
+import io.netty.handler.codec.UnsupportedValueConverter;
+import io.netty.handler.codec.ValueConverter;
+import io.netty.util.internal.UnstableApi;
+
+import static io.netty.util.AsciiString.CASE_INSENSITIVE_HASHER;
+import static io.netty.util.AsciiString.CASE_SENSITIVE_HASHER;
+
+/**
+ * Internal use only!
+ */
+@UnstableApi
```

Comment:
remove and make it package private ?

Reply:
Sure! Ideally though, this should be utils or commons so we can reuse both for http2 and http3 codecs

Reply:
Yep... I will take care and we can cleanup later